File: javascript/2006.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var tipo = parseInt(lines.shift());
var escolhas = lines.shift().trim().split(" ").map(e => parseInt(e));
var corretos = 0;

for(let i = 0; i < 5; i++) {
  if(escolhas[i] ===tipo) {
    corretos++;
  }
}

console.log(corretos);

File: C++/2006.cpp
#include <iostream>

using namespace std;

int main() {
 
    int tipo, opcao, i, corretos = 0;
    cin >> tipo;
    
    for(i = 0; i < 5; i++) {
        cin >> opcao;
        if(opcao == tipo) corretos++;
        
    }
    
    cout << corretos << endl;
    
    return 0;
}